packages/analytics/e2e/analytics.e2e.js

```diff
@@ -1133,5 +1134,28 @@ describe('analytics() modular', function () {
         await setConsent(getAnalytics(), consentSettings);
       });
     });
+
+    describe('getGoogleAnalyticsClientId()', function () {
+      it('Error when trying to use getGoogleAnalyticsClientId() on non-web platforms', async function () {
+        if (!Platform.other) {
+          try {
+            const { getAnalytics } = analyticsModular;
+            await firebase.analytics.getGoogleAnalyticsClientId(getAnalytics());
+            fail('Should have thrown an error');
+          } catch (e) {
+            e.message.should.equal('getGoogleAnalyticsClientId is web-only.');
+          }
+        }
+        this.skip();
+      });
+
+      it('A response is received from getGoogleAnalyticsClientId() on web platforms', async function () {
+        const { getAnalytics } = analyticsModular;
+        if (Platform.OS === 'web') {
+          await firebase.analytics.getGoogleAnalyticsClientId(getAnalytics());
+        }
+        this.skip();
```

Comment:
as mentioned above, I *think* should be:

```suggestion
        } else {
          this.skip();
        }
```

...but I could definitely be missing something

Reply:
The change shouldn't affect anything so happy to commit it.